satpy/tests/writer_tests/test_mitiff.py

```diff
@@ -197,22 +197,22 @@ def test_simple_write(self):
         """Test basic writer operation."""
         from satpy.writers.mitiff import MITIFFWriter
         dataset = self._get_test_dataset()
-        w = MITIFFWriter(mitiff_dir=self.base_dir)
-        w.save_dataset(dataset, writer='mitiff', mitiff_dir=self.base_dir)
+        w = MITIFFWriter(base_dir=self.base_dir)
+        w.save_dataset(dataset, writer='mitiff')
```

Comment:
One more small nitpick, is the `writer=` part needed?

Reply:
Yes, you are right. It's not needed.